### PR TITLE
update(cdk): revamp follow-1

### PR DESCRIPTION
**Pull request by DannyS03**
## Context

Continues necessary improvements for CDK section and the Wiki in general.
- Introduces dedicated redirect file.
- Removes additional references of Supernets.
- Does not showcase Edge in navbar.
- Temp add web3modal to quickstarts
    > These quickstarts will be updated

### CODE REVIEW

File: docs/cdk/validium/what-is-validium.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+id: what-is-validium
+title: What is the CDK Validium?
+sidebar_label: What is Edge
```
Comment (cerberushades) [inline docs/cdk/validium/what-is-validium.md:4]: Sidebar label and title are different. Might that cause some confusion? Are CDK Validium and Edge  the same? If so, we need to clarify this.

File: src/data/features.js [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
+        description: "Learn how you can build next-gen app-chains using the new Polygon chain development kit."
     },
     {
-        title: "Build Custom Solutions",
-        linkUrl: "/docs/category/build-a-supernet",
+        title: "Build Modular & Custom Solutions",
+        linkUrl: "/docs/cdk/validium/quickstart/",
         imageUrl: "img/logo-round-dark.png",
-        description: "Build a customizable PoS app-chain with the Edge consensus client."
+        description: "Build next-gen app-chains using the new Polygon chain development kit."
```
Comment (cerberushades) [inline src/data/features.js:18]: I think we are spelling "appchains" in some places

**Review by cerberushades — CHANGES_REQUESTED** [review]

File: src/data/features.js (hunk 2 above)
Comment (DannyS03) [reply src/data/features.js:18]: I believe we should be using "app-chains" based on the style guide other teams use.

File: docs/cdk/validium/what-is-validium.md (hunk 1 above)
Comment (DannyS03) [reply docs/cdk/validium/what-is-validium.md:4]: Oh meant to delete this file for now - I deleted it.

File: src/data/features.js (hunk 2 above)
Comment (DannyS03) [reply src/data/features.js:18]: Let's address in another PR.

**Review by cerberushades — APPROVED** [review]

**Pull request merged**